dali/operators/image/remap/warp_affine_params.cu

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/remap/warp_affine_params.h"
+#include <cuda_runtime.h>
+
+namespace dali {
+namespace {
+
+template <int ndims>
+__global__ void InvertTransformsK(WarpAffineParams<ndims> *output,
+                                  const WarpAffineParams<ndims> *input, int count) {
+  for (auto i = blockIdx.x * blockDim.x + threadIdx.x; i < count; i += blockDim.x * gridDim.x)
+    output[i] = input[i].invert();
+}
+
+template <int ndims>
+void InvertTransforms(WarpAffineParams<ndims> *output, const WarpAffineParams<ndims> *input,
+                      int count, cudaStream_t stream) {
+  auto blocks = div_ceil(count, 512);
+  auto threads = count < 512 ? count : 512;
```

Comment:
```suggestion
  int blocks = div_ceil(count, 512);
  int threads = min(count, 512);
```
1. `int` is shorter and clearer
2. `min` (global scope) should be available when compiling with NVCC.

Reply:
done